setup.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+from setuptools import setup, find_packages
+
+with open('README.rst', 'rb') as f:
+    LONG_DESCRIPTION = f.read().decode('utf8')
+
+# Single source the version.
+version_file = os.path.realpath(os.path.join(
+    os.path.dirname(__file__), 'pymongoexplain', 'version.py'))
+version = {}
+with open(version_file) as fp:
+    exec(fp.read(), version)
+
+setup(
+    name="pymongoexplain",
+    version=version['__version__'],
+    description="Explainable CRUD API for PyMongo",
+    long_description=LONG_DESCRIPTION,
+    packages=find_packages(exclude=['test']),
+    author="Julius Park",
+    url="https://github.com/mongodb-labs/pymongoexplain",
+    keywords=["mongo", "mongodb", "pymongo"],
+    test_suite="test",
+    tests_require=["pymongo"],
```

Comment:
Please add a `python_requires` section here that indicates what versions of the Python runtime are supported by this package. See https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires for details.